docs/UserGuide.md

```diff
@@ -629,34 +629,27 @@ Format: `list`
 
 ### Finding patients: `find`
 
-Finds patients whose names contain any of the given keywords.
-
-Format: `find [KEYWORD]… [n/NAME]… [p/PHONE]… [e/EMAIL]… [a/ADDRESS]… [t/TAG]… [c/CONDITION]… [d/TASK_DESCRIPTION]… [m/MEDICATION]… [r/REMARK]…`
+Finds patients whose parameters contain the given keywords.
 
+Format: `find [KEYWORD]… [xyz/XYZ_KEYWORD]…`
+* `xyz/XYZ_KEYWORD` refers to a keyword for a specific parameter.
 * There should be at least one parameter for the command.
 * The search is case-insensitive. e.g `hans` will match `Hans`.
 * The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`.
 * Partial words can be matched e.g. `Han` will match `Hans`.
-* Patients matching at least one keyword in every parameter type will be returned (i.e. AND search for different parameters, OR search for same parameter). In more details,
+* Patients matching at least one keyword in every parameter will be returned (i.e. AND search for different parameters, OR search for same parameter). In more details,
     * At least one of the patient's details (name, phone, email, address, tag, condition, task description, or medication) must match with at least one `KEYWORD`.
-    * The patient's name must match at least one `NAME` parameter.
-    * The patient's phone must match at least one `PHONE` parameter.
-    * The patient's email must match at least one `EMAIL` parameter.
-    * The patient's address must match at least one `ADDRESS` parameter.
-    * The patient's tag must match at least one `TAG` parameter.
-    * The patient's condition must match at least one `CONDITION` parameter.
-    * The patient's medication must match at least one `MEDICATION` parameter.
-    * The patient's remarks must match at least one `REMARK` parameter.
+    * For every different parameter `xyz`, it must match at least one `XYZ_KEYWORD`.
```

Comment:
It would refer to the word "parameter", which is defined in the glossary to be the part of a command after a prefix i.e. x/PARAMETER. I think it is more accurate to say a patient's details matches `XYZ_KEYWORD`.
```suggestion
    * For every different parameter `xyz`, the patient's corresponding details must match at least one `XYZ_KEYWORD`.
```